components/klaudsolcms/buttons/AppCreateButton.js

```diff
@@ -1,8 +1,9 @@
 import { FaPlus } from "react-icons/fa";
 import Link from 'next/link';
-const AppCreateButton = ({title, link}) => {
+const AppCreateButton = ({title, link, onClick}) => {
```

Comment:
If there are no statements between `const  AppCreateButton = ( { title, link, onClick } ) => {` (curly braces) and `return (`, you can just omit `return (` and use `const AppCreateButton = ( { title, link, onClick } ) => (` (parentheses) instead.

The code should look like this.

`const AppCreateButton = ( {title, link, onClick} ) => (
  <Link href={link ? link : '/'} passHref>
    <button className="btn_create_entry" onClick={onClick}> 
      <FaPlus className="icon_general"/> 
      {title}
    </button>
  </Link>
)`

Reply:
The code should look like this.

`const AppCreateButton = ( {title, link, onClick} ) => (
  <Link href={link ? link : '/'} passHref>
    <button className="btn_create_entry" onClick={onClick}> 
      <FaPlus className="icon_general"/> 
      {title}
    </button>
  </Link>
)`